package.json

```diff
@@ -168,7 +168,7 @@
     "tslint": "^5.18.0",
     "typescript": "^4.2.3",
     "vscode-test": "^1.2.0",
-    "webpack": "^4.42.0",
+    "webpack": "^5.44.0",
```

Comment:
please notice that the watch `"watch": "webpack --mode development --watch --info-verbosity verbose"` doesn't work in webpack@5 anymore.

It should be `"watch": "webpack --config webpack.config.js --watch"`.

Reply:
good catch, there is a warning telling me "verbose" is no longer effective

Reply:
I'll remove verbosity and upgrade webpack-cli